ttnn/cpp/ttnn/operations/eltwise/unary/unary_composite.hpp

```diff
@@ -119,10 +119,9 @@ struct ExecuteRdiv {
         uint8_t queue_id,
         const Tensor& input_tensor,
         float value,
+        string round_mode = "None",
```

Comment:
* why not std::string (please always specify std:: for stl classes)
* why not const reference (please always pass non primitive types by const &)
* why string type (strings are heap allocated, small-string optimization can allocate on stack, but in this case string is just unreasonable here. keep in mind this has a performance cost)

This can be enum class instead.

Reply:
Sure @ayerofieiev-tt . I will address it.